src/xFactor/base/Application.java

```diff
@@ -0,0 +1,257 @@
+		allParticipants.add(new Participant("Yasen", "Gurvanov", 25, Gender.Male, "Blagoevgrad"));
+		allParticipants.add(new Participant("Grudi", "Radev", 25, Gender.Male, "Stara Zagora"));
+		allParticipants.add(new Participant("Dilyana", "Stolarova", 25, Gender.Female, "Varna"));
+		allParticipants.add(new Participant("Toni", "Ilieva", 25, Gender.Female, "Troyan"));
+		allParticipants.add(new Participant("Zdravko", "Kostadinov", 25, Gender.Male, "Sofia"));
+		allParticipants.add(new Participant("Kristiyan", "Srebrev", 25, Gender.Male, "Purvomay"));
+	}
+
+	private void fillQualities() {
+		qualities = new ArrayList<String>();
+		qualities.add("Dancing");
+		qualities.add("Painting");
+		qualities.add("Playing the guitar");
+		qualities.add("Playing the piano");
+		qualities.add("Playing the viola");
+		qualities.add("Playing the drums");
+	}
+
+	private void initializeStage(int stageNum, int max, Stage stage) {
+		if (stageNum < 1 || stageNum > numberOfStages) {
+			throw new IllegalArgumentException("The number of stage must be between 1 and " + numberOfStages);
+		}
+
+		if (allStages.size() - 1 == numberOfStages) {
+			throw new IllegalArgumentException("Maximum number of stages reached! Cannot create a new one.");
+		} else if (allStages.size() == 1) {
+			stage.setMaxParticipants(allParticipants.size());
+			stage.getParticipantsInStage().addAll(participants);
+		} else {
+			stage.setMaxParticipants(max);
+		}
+	}
+
+	public void createJudges(int number) {
+		judges = new ArrayList<Judge>();
+		addJudges();
+		for (int i = 0; i < number; i++) {
+			judges.add(allJudges.get(i));
+		}
+		judgeDao = new JudgeDao();
+		for (int i = 0; i < judges.size(); i++) {
+			judgeDao.save(judges.get(i));
+		}
+	}
+
+	public ArrayList<Judge> getJudges() {
+		return judges;
+	}
+
+	public void setJudges(ArrayList<Judge> judges) {
+		this.judges = judges;
+	}
+
+	public int getJudgesSize() {
+		return judges.size();
+	}
+
+	public void giveSpecialVote(int id) {
+		if (id > 0 && id < judges.size()) {
+			for (Judge judge : judges) {
+				if (judge.getJudgeId() == id) {
+					judge.setSpecial(true);
+					judgeDao.update(judge);
+				}
+			}
+		}
+	}
+
+	public void printSpecialJudge() {
+		for (Judge judge : judges) {
+			if (judge.isSpecial()) {
+				System.out.println(judge.getName());
+				break;
+			}
+		}
+	}
+
+	public void printJudges() {
+		for (Judge judge : judges) {
+			System.out.println(judge);
+		}
+	}
+
+	public void createParticipants(int number) {
+		participants = new ArrayList<Participant>();
+		addParticipants();
+		for (int i = 0; i < number; i++) {
+			participants.add(allParticipants.get(i));
+		}
+		participantDao = new ParticipantDao();
+		for (int i = 0; i < participants.size(); i++) {
+			participantDao.save(participants.get(i));
+		}
+	}
+
+	public void createStage(int stageNum, int max) {
+		Stage stage = new Stage(stageNum, max);
+		allStages.add(stage);
+		initializeStage(stageNum, max, stage);
+		stageDao = new StageDao();

```

Comment:
I think you may have multiple initialization for one dao, you can do that where you initialize the services